queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -179,10 +208,14 @@ def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-l
 
     # assign subject
     legal_name = business.get('legalName', None)
-    if status == Filing.Status.PAID.value:
-        subject = 'Confirmation of Filing from the Business Registry'
+    if status in [Filing.Status.APPROVED.value, Filing.Status.REJECTED.value]:
+        subject = 'Results of your Filing from the Business Registry'
+    elif status == Filing.Status.CHANGE_REQUESTED.value:
+        subject = 'Change Requested from the Business Registry'
     elif status == Filing.Status.COMPLETED.value:
         subject = 'Continuation Documents from the Business Registry'
+    elif status == [Filing.Status.PAID.value, 'RESUBMITTED']:
```

Comment:
```suggestion
    elif status in [Filing.Status.PAID.value, 'RESUBMITTED']:
```

Reply:
Thanks. I figured this out before I saw your comment. It's fixed in the last commit.